client/app/components/composites/SearchBar/SearchBar.js

```diff
@@ -57,11 +57,17 @@ class SearchBar extends Component {
 
     // Bind `this` within methods
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleResize = this.handleResize.bind(this);
 
-    this.state = { selectedPlace: null };
+    this.state = {
+      selectedPlace: null,
+      mobileMenu: false,
```

Comment:
That's just a flag that toggles if the mobile menu is visible (toggle button clicked). I could rename it to `mobileMenuOpen` if that helps?